platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -460,24 +470,53 @@ public class SwirldsPlatform implements Platform {
 
         components.add(new IssMetrics(platformContext.getMetrics(), currentAddressBook));
 
+        // Manages the pipeline of signed states to be written to disk
+        final SignedStateFileManager signedStateFileManager = new SignedStateFileManager(
+                platformContext,
+                new SignedStateMetrics(platformContext.getMetrics()),
+                Time.getCurrent(),
+                actualMainClassName,
+                selfId,
+                swirldName);
+        // FUTURE WORK: at some point this should be part of the unified platform wiring
+        final WiringModel model = WiringModel.create(platformContext, Time.getCurrent());
```

Comment:
Note that you need to call `model.start()` somewhere, since this is what actually starts the threads used by `SEQUENTIAL_THREAD` schedulers. Adding the model (or the `PlatformWiring` object if you use that instead) to the `components` list will cause it to be started when `platform.start()` is called.

Reply:
yeah, this is why all my tests failed. thanks